components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -182,6 +182,15 @@ open class GleanInternalAPI {
 
         // Set the OS type
         Baseline.os.set("Android")
+        // Set the OS version
+        // https://developer.android.com/reference/android/os/Build.VERSION
+        // TODO: We should probably send the SDK version as well
+        Baseline.osVersion.set(android.os.Build.VERSION.RELEASE)
+        // Set the device string
+        // https://developer.android.com/reference/android/os/Build
+        Baseline.device.set("${android.os.Build.MANUFACTURER} ${android.os.Build.MODEL}")
```

Comment:
The legacy library builds it a bit [differently](https://github.com/mozilla-mobile/android-components/blob/9e2b38f8b98545baaee8d2dd5bb68dd4b9e6b98f/components/service/telemetry/src/main/java/org/mozilla/telemetry/measurement/DeviceMeasurement.java#L26): let's stay equivalent